src/stan-users-guide/efficiency-tuning.qmd

```diff
@@ -1297,39 +1316,40 @@ $$
 \epsilon_n \sim \textsf{normal}(0,\sigma).
 $$
 
-If either vector $x$ or $y$ has very large or very small values or if the
+If vector $x$ has very large or very small values or if the
 sample mean of the values is far away from 0 (on the scale of the values),
-then it can be more efficient to standardize the outputs $y_n$ and
-predictors $x_n$.  The data are first centered by subtracting the
+then it can be more efficient to standardize the
+predictors $x_n$, so that it has a mean of zero and standard deviation of 1.
```

Comment:
This implies standardization of the rows, where `x[m, ]` is a single row.  Instead, we standardize the columns `x[, n]` of the data matrix.

Reply:
this example isn't a data matrix, it's a single predictor.    change to data matrix?